distribution/docs/src/main/resources/_contents/_tables/NSILI_Federated_Source-table-contents.adoc

```diff
@@ -0,0 +1,222 @@
+.[[NSILI_Federated_Source]]NSILI Federated Source
+[cols="1,1m,1,3,1,1" options="header"]
+|===
+
+|Name
+|Id
+|Type
+|Description
+|Default Value
+|Required
+
+|NSILI Source ID
+|id
+|String
+|The ID of the NSILI Source
+|NSILI Federated Source
+|true
+
+| Ior File URL
+| iorUrl
+| String
+| The URL of the IOR file to use for the NSILI CORBA Service. Supported URL protocols are 'http://'
+| null
+| true
+
+| HTTP/S or FTP Username
+| serverUsername
+| String
+| 'https://'
+| null
+| false
+
+| HTTP/S or FTP Password
+| serverPassword
+| Password
+| 'ftp://'
+| 
+| false
+
+| HTTP/S or FTP Timeout
+| clientTimeout
+| Integer
+| and 'file://'.
+| 60
+| true
+
+| Max Hit Count
+| maxHitCount
+| Integer
+| The Username to be used for authentication with HTTP or FTP server.
+| 250
+| true
+
+| Additional Query Params
+| additionalQueryParams
+| String
+| The Password to be used for authentication with HTTP or FTP server.
+| and (not NSIL_PRODUCT:NSIL_CARD.status = 'OBSOLETE')
+| false
+
+| Poll Interval
+| pollInterval
+| Integer
+| The timeout (seconds) associated with retrieving the IOR file.
+| 5
+| true
+
+| Num Worker Threads
+| numberWorkerThreads
+| Integer
+| The Maximum Hit Count for Queries to the Source
+| 4
+| true
+
+| Exclude Sort Order
+| excludeSortOrder
+| Boolean
+| NSILI Query Parameters to always include
+| false
+| false
+
+| Swap X/Y Coordinates
+| swapCoordinates
+| Boolean
+| Poll Interval to Check if the Source is available (in minutes - minimum 1).
+| false
+| false
+
+| NSILI Access Criteria User
+| accessUserId
+| String
+| Maximum parallel threads for converting results and retrieving thumbnails
+| 
+| false
+
+| NSILI Access Criteria Password
+| accessPassword
+| Password
+| Whether or not to exclude sort order in query.
+| 
+| false
+
+| NSILI Access Criteria License Key
+| accessLicenseKey
+| String
+| Swap coordinates for systems that send latitude in the X field of the coordinate.
+| 
+| false
+
+|===
+
+.[[NSILI_Connected_Source]]NSILI Connected Source
```

Comment:
@ricklarsen did you push up the updated tables? @peterhuffer's comment still seems valid based on the latest on the PR.